tutorials/common/common_utils.cpp

```diff
@@ -199,11 +212,13 @@ mfxStatus LoadRawRGBFrame(mfxFrameSurface1* pSurface, FILE* fSource)
 
 mfxStatus WriteBitStreamFrame(mfxBitstream* pMfxBitstream, FILE* fSink)
 {
-    mfxU32 nBytesWritten =
-        (mfxU32) fwrite(pMfxBitstream->Data + pMfxBitstream->DataOffset, 1,
-                        pMfxBitstream->DataLength, fSink);
-    if (nBytesWritten != pMfxBitstream->DataLength)
-        return MFX_ERR_UNDEFINED_BEHAVIOR;
+    if (fSink) {
```

Comment:
I don't think it is correct solution. I vote for

```
if (!pMfxBitstream)
   return MFX_ERR_NULL_PTR;

if (!fSink)
   return MFX_ERR_INVALID_HANDLE;
```

Reply:
We can have fSink == nullptr, if run simple_* in non-output mode...